### PR TITLE
enh(UI): PENDO - Add unique identifier for css objects in rs

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

Add unique identifier on css objects in resource status

- ACK / DT / Check buttons

- ACK / DT / Check icons on each resources line

- Url / Action note icons on each resources line

- Clear filter icon

- Manage filters icon

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Go on each element , click right then inspect on <button/> or <div/> or <span/> , you can see attribute data-testid between tha balise on HTML element

### CODE REVIEW

File: www/front_src/src/Resources/Actions/Refresh/index.tsx [hunk 1]
```diff
@@ -69,6 +69,7 @@ const RefreshActions = ({ onRefresh }: Props): JSX.Element => {
       <Grid item>
         <IconButton
           ariaLabel={t(labelRefresh)}
+          data-testid={t('Refresh')}
```
Comment (bdauria) [inline www/front_src/src/Resources/Actions/Refresh/index.tsx:72]:
```suggestion
          data-testid={t(labelRefresh)}
```

Comment (Jordan91) [reply www/front_src/src/Resources/Actions/Refresh/index.tsx:72]: oups ^^

**Review by Thebarda — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]
The code seems ok to me, but I don't like the term `data-testid`. I would have liked to have a more descriptive term.
example : `data-pendoid`

**Comment by Jordan91** [comment]
Commits cherry-pick on another PR - JCA

**Pull request closed without merging**